.azure-pipelines/util/test-steps.yml

```diff
@@ -16,6 +16,11 @@ steps:
     custom: msbuild
     arguments: 'build.proj /t:${{ parameters.testTarget }} /p:Configuration=${{ parameters.configuration }};TestFramework=${{ parameters.testFramework }};PullRequestNumber=$(System.PullRequest.PullRequestNumber)'
 
+- task: PublishTestResults@2
+  inputs:
+    testRunner: VSTest
+    testResultsFiles: '**/*.trx'
```

Comment:
@dingmeng-xue, based on https://github.com/Azure/azure-powershell/blob/master/build.proj#L246, it may be
```
testResultsFiles: '**/trx'
```

Reply:
trx is suffix of log file instead of full name. "TestResults\VssAdministrator_fv-az379_2019-10-23_09_08_01.trx" is one example.
*.trx should be correct.